components/Chat/Message/MessageReactions/MessageReactionsDrawer/MessageReactionsDrawer.tsx

```diff
@@ -164,6 +164,7 @@ const useStyles = () => {
       alignItems: "flex-end",
       gap: theme.spacing.xxxs,
       overflow: "hidden",
+      color: theme.colors.text.primary,
```

Comment:
I think we should use the `color` prop and preset as much as possible. If it's really unique let's not create new preset but usually Andrew should reuse a lot of presets so

Reply:
Same issue as the reaction menu title appears black on back in dark mode on Android Galaxy A05S 👀 Although it does not seem to be the case anymore...

Reply:
I've migrated to $ styles here: https://github.com/ephemeraHQ/converse-app/pull/1209
I'll make sure to check presets and update accordingly.